pkg/server/grpc/tracee.go

```diff
@@ -19,6 +19,456 @@ import (
 	"github.com/aquasecurity/tracee/types/trace"
 )
 
+// Syscall table map - internal syscall ID and external ID
+var syscallTranslationMap = map[events.ID]pb.Id{
```

Comment:
@geyslan and I were discuss about using a array instead of mapping here, to be more performatic... and for each arch have one array, WDYT @yanivagman

Reply:
Maybe one array would fit both archs range. The array index being the `syscall number` and its value being our `pb.Id`.

In this scenario, the array would only be filled at initialization.

Reply:
good idea

Reply:
> Maybe one array would fit both archs range. The array index being the `syscall number` and its value being our `pb.Id`.
> 
> In this scenario, the array would only be filled at initialization - based on the arch.

We previously used build tags for this. So we can use the already existing code for that in `pkg/events/core/core_<arch>.go`, and refactor it into the array.